fighthealthinsurance/medicaid_api.py

```
@@ -0,0 +1,931 @@
+from pathlib import Path
+from typing import Any, Dict, Tuple, List, Optional, Sequence
```

Ruff (0.12.2)

`typing.Sequence` imported but unused

Remove unused import: `typing.Sequence`

(F401)